test/modules/losses/test_commitment.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import unittest
+
+import torch
+from torchmultimodal.modules.losses.vqvae import CommitmentLoss
+
+
+class TestCommitment(unittest.TestCase):
+    """
+    Test the Commitment Loss
+    """
+
+    def setUp(self):
+        torch.set_printoptions(precision=10)
+        torch.manual_seed(4)
+        self.quantized = torch.randn((2, 3))
+        self.encoded = torch.randn((2, 3))
+
+    def test_loss_value(self):
+        commitment = CommitmentLoss()
+        loss = commitment(self.quantized, self.encoded)
+
+        actual = loss.loss.item()
+        expected = 1.2070025206
```

Comment:
Personally I prefer manual inputs and computation of the expected for simple logic like L2 norm. Otherwise, it just becomes a regression test (checks if future values deviate from the current snapshot) rather than unit test (which checks for correctness).

Reply:
not sure what you mean by manual, do you mean using hand-picked values instead of random values with a fixed rng seed?